chart/compass/templates/test-definitions/external-services-mock-test.yaml

```diff
@@ -2,21 +2,21 @@
 apiVersion: "testing.kyma-project.io/v1alpha1"
 kind: TestDefinition
 metadata:
-  name: {{ .Chart.Name }}-gateway-external-services-integration
+  name: {{ .Chart.Name }}-e2e-external-services-mock
```

Comment:
The convention that we are trying to enforce is compass-e2e-"component-name". It will be inconsistent with the other components if the name here is compass-e2e-external-services-integration